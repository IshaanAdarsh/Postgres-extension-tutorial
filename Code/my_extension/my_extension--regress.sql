-- regression test script for my_extension

-- Create the my_table table
CREATE TABLE my_table (
  id SERIAL PRIMARY KEY,
  name VARCHAR(100) NOT NULL
);

-- Test the my_function function
SELECT my_function();

-- Verify the data in the my_table table
SELECT * FROM my_table;
