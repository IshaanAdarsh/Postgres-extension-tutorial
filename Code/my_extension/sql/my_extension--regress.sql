-- regression test script for my_extension

-- Create the extension
CREATE EXTENSION my_extension;

-- Verify the data in the my_table table
SELECT * FROM my_table;

-- Test the add function
SELECT add(1, 2) AS result;

-- Test the complex_add function
SELECT complex_add(ARRAY[1, 2, 3, 4, 5]) AS result;

-- Test the multiply function
SELECT multiply(3, 4) AS result;
