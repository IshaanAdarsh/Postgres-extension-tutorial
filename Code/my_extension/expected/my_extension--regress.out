 id |   name   
----+----------
  1 | Example
